onnxruntime/core/providers/iree/compiler/torch-mlir-import-onnx/OnnxImporter.cpp

```diff
@@ -613,15 +639,15 @@ void NodeImporter::PopulateGraphAttrs(MlirOperation container_op) {
       mlirStringAttrGet(context_, toMlirStringRef(m.producer_version())));
 }
 
-Status NodeImporter::ImportAll() {
+Status NodeImporter::ImportAll(const onnxruntime::GraphViewer &gv) {
   // TODO: Consider pulling in initializers on demand since there can be so
   // much unused crap.
   for (auto it : graph_info_.initializer_map()) {
     if (failed(ImportInitializer(it.second)))
       return failure();
   }
   for (auto it : graph_info_.graph_proto().node()) {
-    if (failed(ImportNode(it)))
+    if (failed(ImportNode(it, gv)))
```

Comment:
Is there a way to report what is failing for easier debugging?

Reply:
The errors should propagate through the function call stack. This the errors are usually reported at the last point of failure. If we report errors at each function in the error path, there will be too many messages and the actual problem might get lost 
Your thoughts?